components/prize_pool/commons/prize_pool.lua

```diff
@@ -77,7 +78,24 @@ PrizePool.prizeTypes = {
 	[PRIZE_TYPE_LOCAL_CURRENCY] = {
 		header = 'localcurrency',
 		headerParse = function (prizePool, input, context, index)
-			return {currency = string.upper(input)}
+			Variables.varDefine('localcurrencysymbol', '')
+			Variables.varDefine('localcurrencysymbolafter', '')
+			Variables.varDefine('localcurrencycode', '')
+			local currencyText = Template.safeExpand(mw.getCurrentFrame(), 'Local currency', {input})
+
+			local symbol, symbolFirst
+			if String.isNotEmpty(Variables.varDefault('localcurrencysymbol')) then
+				symbol = Variables.varDefault('localcurrencysymbol')
```

Comment:
What's the point of using `varDefault` like this?

Reply:
It will return `nil` instead of the empty string. Allowing the `String.isNotEmpty` to be skipped (which I did here initially anyway - but removed now)